region/lowernorfair/east.json

```diff
@@ -2702,51 +2708,136 @@
+                  {"cost":[
+                    {"enemyDamage": [{
+                      "enemy": "Dessgeega",
+                      "type": "contact",
+                      "hits": 1
+                    }]}
+                  ]}
+                ]},
+                {"and":[
+                  {"obstacle":{
+                    "id": "C",
+                    "requires": [
+                      {"obstacle":{
+                        "id": "E",
+                        "requires": null
+                      }},
+                      {"or":[
+                        {"and":[
+                          "Plasma",
+                          {"cost":[ {"heatFrames": 200}]},
+                          {"or":[
+                            "Wave",
+                            {"cost":[ {"heatFrames": 200}]}
+                          ]}
+                        ]},
+                        {"and":[
+                          "Missile",
+                          {"cost":[ {"heatFrames": 400}]}
+                        ]},
+                        {"and":[
+                          "Super",
+                          {"cost":[ {"heatFrames": 400}]}
+                        ]}
+                      ]}
+                    ]
+                  }}
+                ]},
                 {"obstacle":{
-                  "id": "D",
+                  "id": "C",
                   "requires": [
-                    {"or": [
-                      "Plasma",
-                      "Missile",
-                      "Super",
-                      "ScrewAttack"
+                    {"or":[
+                      {"and":[
+                        "Plasma",
+                        {"cost":[ {"heatFrames": 400}]},
+                        {"or":[
+                          "Wave",
+                          {"cost":[ {"heatFrames": 200}]}
+                        ]}
+                      ]},
+                      {"and":[
+                        "Missile",
+                        {"cost":[ {"heatFrames": 400}]}
+                      ]},
+                      {"and":[
+                        "Super",
+                        {"cost":[ {"heatFrames": 200}]}
+                      ]},
+                      {"and":[
+                        "ScrewAttack",
+                        {"cost":[ {"heatFrames": 50}]}
+                      ]}
                     ]}
                   ]
-                }},
-                {"roomReset":{
-                  "nodes": [1],
-                  "obstaclesToAvoid": ["E"]
                 }}
               ],
               "unlock": null,
-              "note": "The clip is done by getting the far right Multiviola in the bottom hallway to climb all the way up. It requires several trips through the hallway, hence killing the Dessgeegas"
+              "note": "Options are to Speed Booster dash or spark through, tank the damage, kill the Dessgeegas and Multiviolas, or go slower to spare the Multiviolas"
```

Comment:
The Plasma option is listed as slower without killing Violas while the Super option is slower not killing them. Might make sense I guess. Also there's no ScrewAttack option for not killing violas which might also make sense. All of the sense is just hidden very deep down in this room.

Reply:
The Supers thing sounds wrong. Pretty sure the time should be the same. I got tripped up counting damage due to sometimes forgetting about the 50 damage from the Multiviola. I'll have to recount that.

Not sure if you mistyped, but there is a Screw Attack option for not killing the Multiviolas. I left out the option for killing them because you can't really catch up to them before reaching node 7.

Plasma is much slower without killing the Multiviolas because the beams pierce and you have to wait till it despawns.